setup/jetstream/nats.go

```diff
@@ -0,0 +1,93 @@
+package jetstream
+
+import (
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/Shopify/sarama"
```

Comment:
Key change consumers still use `github.com/S7evinK/saramajetstream` because they expect a partition number as well as an offset. They will need to be rewritten for NATS nativeness.

Reply:
Okay we should replace this ASAP then - can we do this next?